### PR TITLE
test(IsTheSqaureAchild):add test & solution

### PR DESCRIPTION
**Pull request by Hamzaelkhatri**

### CODE REVIEW

**Review by zainabdnaya — APPROVED** [review]
All Good

File: solutions/isthesquareachild.go [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package solutions
+
+func IsTheSquareAChild(number int, square int) bool {
+	if number < 0 || square < 0 {
+		return false
+	}
+	return number*number == square
```
[the remark this answers is not in the document]
Comment (Hamzaelkhatri) [reply solutions/isthesquareachild.go:7]: i change the way of return , in the issue @mohamedLazyBob  say `(9, 3) => true`
n1 * n1 == n2 or n2 * n2 == n1 . but i use just the first case , to make the execices more easier

File: tests/isthesquareachild_test/main.go [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+	"github.com/01-edu/go-tests/lib/challenge"
+	"github.com/01-edu/go-tests/lib/random"
+	"github.com/01-edu/go-tests/solutions"
+)
+
+func main() {
+
+	args:= [][]int{
+		{1, 1},
```
Comment (MSilva95) [inline tests/isthesquareachild_test/main.go:14]: add the case that @zainabdnaya asked in the readme file ...

**Review by MSilva95 — CHANGES_REQUESTED** [review]

**Review by zainabdnaya — APPROVED** [review]
LGTM

File: solutions/issquare.go [hunk 3]
```diff
@@ -0,0 +1,8 @@
+package solutions
+
+func isSquare(number int, square int) bool {
```
Comment (MSilva95) [inline solutions/issquare.go:3]: function name is wrong...
Needs to be upercase.
IsSquare()

**Review by MSilva95 — CHANGES_REQUESTED** [review]

File: solutions/issquare.go (hunk 3 above)
Comment (Hamzaelkhatri) [reply solutions/issquare.go]: fixed

File: tests/issquare_test/main.go [hunk 4]
```diff
@@ -0,0 +1,26 @@
+package main
+
+import (
+	student "student"
+
+	"github.com/01-edu/go-tests/lib/challenge"
+	"github.com/01-edu/go-tests/solutions"
+)
+
+func main() {
```
Comment (MSilva95) [inline tests/issquare_test/main.go:10]: add more tests, i can pass this with a wrong solution like so.
```
func IsSquare(number int, square int) bool {
	return number*number == square
}
```

**Review by MSilva95 — CHANGES_REQUESTED** [review]

File: tests/issquare_test/main.go (hunk 4 above)
Comment (Hamzaelkhatri) [reply tests/issquare_test/main.go:10]: fixed

**Review by MSilva95 — APPROVED** [review]

**Pull request merged**